Cue/app/actions/types.js

```diff
@@ -20,7 +20,8 @@ export type Action =
 	| { type: 'DECK_ALREADY_IN_LIBRARY'}
 	| { type: 'DECK_ADDED_TO_LIBRARY', addedDeck: Deck}
 	| { type: 'CLEAR_INACCESSIBLE_DECKS'}
-;
+	| { type: 'CARD_RATED', change: {uuid: string, userRating: number}}
```

Comment:
No, I was just wondering why you pass in a `change` object instead of just passing in `uuid` and `userRating` separately, but I guess this makes sense.

This should be `DECK_RATED`, instead of `CARD_RATED`, right?